source/mesh_deformation/fastscape.cc

```diff
@@ -1867,7 +1872,8 @@ namespace aspect
 
             prm.declare_entry("Sea level", "0.0",
                               Patterns::Double(),
-                              "Constant sea level relative to the ASPECT surface, where the maximum Z or Y extent in ASPECT is a sea level of zero. Units: $\\{m}$ ");
+                              "Constant sea level relative to the ASPECT surface, where the maximum Z or Y extent in ASPECT is a sea level of zero. "
+                              "Units: $\\{m}$ ");
```

Comment:
I suspect that the double backslash is wrong here. Let's fix this in a follow-up.